docs/UserGuide.md

```diff
@@ -310,156 +310,156 @@ the constraints of each parameter when used in a command.
-> * **At least one** of the optional fields must be provided. e.g. `edit S1234567A` is invalid.
+> * The NRIC provided must be the full NRIC of the patient to be edited. 
+>   - e.g. `S1234567A` and not `S123`.
+> * **At least one** of the optional fields must be provided. 
+>   - e.g. `edit S1234567A` is invalid.
 > * Existing values will be updated to the given input values.
-> * You can edit a patient's details even if they are not being currently displayed in the Patient List Panel but doing so will refresh the panel to display all patients after the patient's details have been edited.
-> * Refer to the [Parameter Details](#parameter-details) section for more information on the purpose and constraints of each parameter.
+> * You can edit a patient's details even if they’re not currently displayed in the Patient List Panel but doing so will refresh the panel to display all patients after the patient's details have been edited.
+> * Refer to [Parameter Details](#parameter-details) for constraints on each parameter.
 
-Example:
+**Example**:
 *  `edit S1234567A p/91234567 e/johndoe@example.com` will edit the phone number and email address of the patient with the NRIC`S1234567A`
 to `91234567` and `johndoe@example.com` respectively.
 
 {: .alert .alert-success}
 > :bulb: **Tip:**
 > 
-> Editing the patient's `NAME` or `NRIC` will also update their associated appointments in the Appointment List Panel to reflect the new change.
+> Editing the patient's `NAME` or `NRIC` will also update their associated appointments in the Appointment List Panel to reflect the change.
 
 [Back to Table of Contents](#table-of-contents)
 
 <div style="page-break-after: always;"></div>
 
 ### Managing Appointments
 
-#### Adding Appointments : `addAppt`
+#### Adding an Appointment : `addAppt`
 
-Allow doctors to schedule an appointment with a patient.
+Schedules an appointment for a patient in MediBase3.
 
-Format: `addAppt APPOINTMENT_NAME i/NRIC @d/APPOINTMENT_DATE @t/APPOINTMENT_TIME`
+**Format**: `addAppt APPOINTMENT_NAME i/NRIC @d/APPOINTMENT_DATE @t/APPOINTMENT_TIME`
 
 {: .alert .alert-info}
 > :information_source: **Note:**
 >
-> * Adds Appointment Time and Appointment Name to the patient with the specified `NRIC` in MediBase3.
-> * You can schedule an appointment to a patient even if they are not being currently displayed in the Patient List Panel but doing so will refresh the panel to display all patients after their appointment has been added.
-> * Refer to the [Parameter Details](#parameter-details) section for more information on the purpose and constraints of each parameter.
+> * Adds an appointment with the specified `APPOINTMENT_NAME`, `APPOINTMENT_DATE`, and `APPOINTMENT_TIME` for the patient identified by `NRIC`.
+> * If the patient is not currently displayed in the Patient List Panel, the panel will refresh to show all patients after adding the appointment.
```

Comment:
Should maintain consistency for such 'panel altering' message as it is use for every command

![image](https://github.com/user-attachments/assets/7572bf2d-54be-4d4d-8f34-77deca036815)